client/signup/steps/domains/index.jsx

```diff
@@ -478,7 +478,7 @@ class DomainsStep extends React.Component {
 				isEligibleVariantForDomainTest={ this.isEligibleVariantForDomainTest() }
 				suggestion={ initialQuery }
 				designType={ this.getDesignType() }
-				vendor={ getSuggestionsVendor( true, this.props.userGeoCountryCode ) }
+				vendor={ getSuggestionsVendor( true ) }
```

Comment:
We can also remove `userGeoCountryCode` from props, right? Otherwise it will still fetch geolocation data.